app/main/helpers/validation.py

```diff
@@ -282,6 +282,37 @@ def errors(self):
         return errors_map
 
 
+class DOS5Validator(SharedValidator):
+    """Following an accessibility review, a number of questions and answers were changed for DOS 5"""
+    email_validation_fields = {"contactEmailContractNotice", "contactEmail"}
+    percentage_fields = ["subcontractingInvoicesPaid"]
+
+    def is_valid_percentage(self, field):
+        value = self.answers.get(field)
+
+        # Design System guidance is that we should allow users to provide answers with or without units.
+        if isinstance(value, str):
+            value = value.rstrip('%')
+
+        try:
+            number = float(value)
+        except ValueError:
+            return False
+        else:
+            return 0 <= number <= 100
+
+    def formatting_errors(self, answers):
+        error_map = super(DOS5Validator, self).formatting_errors(answers)
+
+        error_map.update({
+            field: 'invalid_format'
```

Comment:
I think this needs to be 'not_a_number'? [[1]](https://github.com/alphagov/digitalmarketplace-frameworks/blob/master/frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontractingInvoicesPaid.yml).

Currently I'm getting the error message 'You must enter a valid email address.'

![You must enter a valid email address.](https://user-images.githubusercontent.com/503614/94800674-d23f8a00-03dc-11eb-966f-f39844d2b75f.png)